### PR TITLE
Increase kube-controller-manager VPA minAllowed

### PR DESCRIPTION
**Pull request by ggaurav10**
**How to categorize this PR?**

/area control-plane
/area auto-scaling
/priority normal

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
Increase kube-controller-manager VPA minAllowed
```

### CODE REVIEW

**Comment by rfranzke** [comment]
/invite @timebertt 
CLA needs to be signed.

**Review by rfranzke — APPROVED** [review]
/lgtm
/needs second-opinion

File: pkg/operation/botanist/controlplane/kubecontrollermanager/kube_controller_manager.go [hunk 1]
```diff
@@ -302,7 +302,7 @@ func (k *kubeControllerManager) Deploy(ctx context.Context) error {
 			ContainerPolicies: []autoscalingv1beta2.ContainerResourcePolicy{{
 				ContainerName: kubeControllerManagerContainerName,
 				MinAllowed: corev1.ResourceList{
-					corev1.ResourceCPU:    resource.MustParse("25m"),
+					corev1.ResourceCPU:    resource.MustParse("100m"),
```
Comment (timebertt) [inline pkg/operation/botanist/controlplane/kubecontrollermanager/kube_controller_manager.go:305]: FYI: `25m` was used based on the suggestion in https://github.com/gardener/gardener/pull/3057#discussion_r513640869 (cc @amshuman-kr)
But I'm fine with increasing it again to stabilize the control planes.

**Review by timebertt — APPROVED** [review]
/lgtm

**Pull request merged**